src/report/ReportRecordProcessing.tsx

```diff
@@ -29,6 +29,24 @@ export function extractNodePropertiesFromRecords(records: any) {
   return fields.length > 0 ? fields : [];
 }
 
+/**
+ * Collects all node labels and node properties in a set of Neo4j records.
+ * @param records : a list of Neo4j records.
+ * @returns a list of lists, where each inner list is [NodeLabel] + [prop1, prop2, prop3]...
+ */
+export function extractNodeRelPropertiesFromRecords(records: any) {
```

Comment:
`NodeAndRelProperties` instead of `NodeRelProperties`?

Reply:
Done